src/printer/smt2/smt2_printer.cpp

```diff
@@ -59,7 +59,48 @@ bool isVariant_2_6(Variant v) { return v == smt2_6_variant; }
 static void toStreamRational(std::ostream& out,
                              const Rational& r,
                              bool decimal,
-                             Variant v);
+                             Variant v)
+{
+  bool neg = r.sgn() < 0;
+  // Print the rational, possibly as decimal.
+  // Notice that we print (/ (- 5) 3) instead of (- (/ 5 3)),
+  // the former is compliant with real values in the smt lib standard.
+  if (r.isIntegral())
+  {
+    if (neg)
+    {
+      out << (v == sygus_variant ? "-" : "(- ") << -r;
```

Comment:
Could you write a comment here as well like sygus requires `-n` instead of `(- n)` for negative integers.

Reply:
This is outdated actually, ill remove.